File: Soal-1/PRAK501-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>

int MaxBilangan(int a, int b, int c, int d) {
    int hasil = a;
    if (b > hasil){
        hasil = b;
    }
    if (c > hasil){
        hasil = c;
    }
    if (d > hasil){
        hasil = d;
    }

    return hasil;
}

int main() {
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);
    return 0;
}

File: Soal-1/PRAK501-2410817310014-INDRA_SURYADILAGA.py
def MaxBilangan(a, b, c, d):
  hasil = a
  if b > hasil:
    hasil = b
  if c > hasil:
    hasil = c
  if d > hasil:
    hasil = d
  return hasil

a, b, c, d = map(int, input().split())
hasil = int(MaxBilangan(a, b, c, d))
print(hasil)
